src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -452,12 +452,21 @@ def fingerprint_span_op(span: Span):
     return op
 
 
-def get_span_duration(span: Span):
+def get_span_duration(span: Span) -> timedelta:
     return timedelta(seconds=span.get("timestamp", 0)) - timedelta(
         seconds=span.get("start_timestamp", 0)
     )
 
 
+def contains_complete_query(span: Span, is_source: Optional[bool] = False) -> bool:
+    # Remove the truncation check from the n_plus_one db detector.
+    query = span.get("description", None)
+    if is_source and query:
```

Comment:
I forgot about the `is_source` parameter on this - bit awkward now that this is pulled out of context but we can refactor later

Reply:
Will make a note of this